pkg/metrics/metrics.go

```diff
@@ -17,7 +17,7 @@ var (
 
 	metricRequestsByService = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
-			Name: "ocm_agent_requests_by_service",
+			Name: "ocm_agent_requests_by_service", // this is a counter without the _total suffix. What about renaming metric to ocm_agent_requests_by_service_total ?
```

Comment:
Would you like to apply all the changes on the metrics naming suggestion? We can review after that.
But I think if we change the metrics name, we should update all the existing references, like prometheusrules if there is any.
And I remember we have added those metrics to the grafana dashboard, not sure if any update there needed.

Reply:
> Would you like to apply all the changes on the metrics naming suggestion?

Maybe, but not in this pull request as the only purpose of this PR is to add the place holder feature to `ocm-agent`

> But I think if we change the metrics name, we should update all the existing references, like prometheusrules if there is any.
And I remember we have added those metrics to the grafana dashboard, not sure if any update there needed.

Indeed, that's quite some work... also the `PrometheusRule`s and dashboards will probably have to be modified to accommodate both old & new names for a while.
So, to be honest, I don't know if it would be that much of a great idea to rename those metrics.
However I think this should be done in a separate pull request to not clutter and delay this PR which is quite urgent.
But before even submitting a PR for this change, I think it would be best to create a ticket and have the various contributors (team leads, team members, func lead) debate over the necessity of this change. @bmeng : WDYT?
My comments about the names of those metrics was more to open the debate :) ... and I can remove them if we create a ticket to track this renaming question.

Reply:
+1 to put them into separated PR if required
Feel free to create a card to discuss with the owners firstly I would suggest

Reply:
@bmeng: I created a separate ticket for that: [OSD-19707](https://issues.redhat.com/browse/OSD-19707) :-)